client/python/modeldb/basic/ModelDbSyncerBase.py

```diff
@@ -203,9 +204,7 @@ def clear_buffer(self):
     def initialize_thrift_client(self, thrift_config):
         # use defaults if thrift_config values are empty
         if not (thrift_config.port and thrift_config.host):
-            syncer_location = os.path.abspath(
-                os.path.join(os.path.dirname(__file__), self.config_file))
-            config_reader = ConfigReader(syncer_location)
+            config_reader = ConfigReader(self.config_path)
```

Comment:
This works - config_path is set as a class variable - since it's a class method, self refers to the class. It'd be good to set that explicitly though, so will update.